app/views/admin/event_teammate/index.html.haml

```diff
@@ -1,27 +1,27 @@
 %h1= event
```

Comment:
Is this even used?  Admins don't do anything with teammates so this can be removed assuming it's not used.